### PR TITLE
Update operations according to the new beta

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

File: core/operations.md [hunk 1]
```diff
@@ -258,12 +258,17 @@ class BookSpecial
...
+    public function __invoke($data) // API Platform retrieves the PHP entity using the data provider then (for POST and
+                                    // PUT method) deserializes user data in it. Then passes it to the action. Here $data
+                                    // is an instance of Book having the given ID. By convention, the action's parameter
+                                    // must be called $data.
     {
-        // do something special here
+        $this->myService->doSomething($data);
 
-        return $this->doctrine->getManager()->find(Book::class, $id);
+        return $data; // API Platform will automatically validate, persist (if you use Doctrine) and serialize en entity
```
Comment (soyuka) [inline core/operations.md]: an

File: core/operations.md [hunk 2]
```diff
@@ -278,14 +283,17 @@ together.
 Here we consider that DunglasActionBundle is installed (the default when using the API Platform standard edition). This
 action will be automatically registered as a service (the service name is the same as the class name: `AppBundle\Action\BookSpecial`).
 
-The Doctrine manager registry (`doctrine` service) will be automatically injected thanks to the autowiring feature. You
-can type-hint any other service you need and it will be autowired too.
+API Platform automatically retrieve the appropriate PHP entity then deserializes
```
Comment (soyuka) [inline core/operations.md]: then deserializes it

File: core/operations.md [hunk 3]
```diff
@@ -278,14 +283,17 @@ together.
...
+
+Services (`$myService` here) are automatically injected thanks to the autowiring feature. You can type-hint any service
+you need and it will be autowired too.
```
Comment (soyuka) [inline core/operations.md]: Maybe link to autowiring docs?

Comment (dunglas) [inline core/operations.md]: There is already a link in the top of this document.

**Pull request merged**